docs/sphinx/releases/whats_new_1_4_0.rst

```diff
@@ -14,11 +14,59 @@
 General changes
 ===============
 
+* We have added the collectives ``all_to_all`` and ``all_reduce``.
+* Components can now be checkpointed.
+* We have added the performance counters ``background-receive-duration`` and
+  ``background-receive-overhead`` for inspecting the time and overhead spent on
+  receiving parcels in the backgroun d.
```

Comment:
There is a stray space in the last word.

Reply:
Good catch!